InnoClinic.ProfilesAPI/Domain/Entities/Receptionist.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Domain.Entities;
+
+public sealed class Receptionist
+{
+    public Guid Id { get; set; }
+    public string FirstName { get; set; } = string.Empty;
+    public string LastName { get; set; } = string.Empty;
+    public string? MiddleName { get; set; }
```

Comment:
These fields are the same as for others. Why not to have a class for duplicated fields?